releasenotes/releasenotes_develop.md

```diff
@@ -1 +1,123 @@
+- Derive the list of deployments that are currently running for a service [3629](https://github.com/keptn/keptn/issues/3629)
+- Uniform names of mock files [3714](https://github.com/keptn/keptn/issues/3714)
+- Add "load older Sequences" button in Sequence screen [2280](https://github.com/keptn/keptn/issues/2280)
+- Sequence icon colors represent status [3591](https://github.com/keptn/keptn/issues/3591)
+- Show target values for criteria when hovering over values [2757](https://github.com/keptn/keptn/issues/2757)
+- *Fixed*: Chart in Heatmap shows incorrect Y-axis and incorrect tooltip [3645](https://github.com/keptn/keptn/issues/3645)
+- *Fixed*: Daily version check was disabled [3668](https://github.com/keptn/keptn/issues/3668)
+- *Fixed*: Quality gate icon in the environment screen does not turn red [3592](https://github.com/keptn/keptn/issues/3592)
+- *Fixed*: Some deep-links are broken [3631](https://github.com/keptn/keptn/issues/3631)
+- *Fixed*: Problem filter in environment screen does not work [3652](https://github.com/keptn/keptn/issues/3652)
+
+</p>
+</details>
+
+## Miscellaneous
+
+- Delete helm-service and jmeter-service from continuous-delivery Helm Chart and adapt CLI accordingly [3350](https://github.com/keptn/keptn/issues/3350)
+- Decouple unit tests from "get.keptn.sh/version.json" [3476](https://github.com/keptn/keptn/issues/3476)
+
+## Development Process / Testing
+
+- GitHub Action to setup a Keptn deployment [3554](https://github.com/keptn/keptn/issues/3554)
+- GitHub Action to send CloudEvents to a Keptn installation [2797](https://github.com/keptn/keptn/issues/2797)
+- Use github.com/stretchr/testify/require instead of general asserts [3149](https://github.com/keptn/keptn/issues/3149)
+
+## Good to know / Known Limitations
+
+- See the know limitations from [0.8.0](https://github.com/keptn/keptn/releases/tag/0.8.0)
+
+<details><summary>Open issues that will be fixed in upcoming releases</summary>
+<p>
+
+  <!--TODO: final check-->
+  - Auto-remediation does not work with remote execution plane [3498](https://github.com/keptn/keptn/issues/3498)
+  - Bridge shows "started" wording on status.changed [3583](https://github.com/keptn/keptn/issues/3583)
+  - Inconsistent usage of user-managed and user_managed causing issues [3624](https://github.com/keptn/keptn/issues/3624)
+  - Keptn API: `configure monitoring` not functioning according to spec [3638](https://github.com/keptn/keptn/issues/3638)
+  - Keptn CLI: Disable Kube context check [3666](https://github.com/keptn/keptn/issues/3666)
+  - Homebrew installed CLI fails install with 'Malformed constraint: ""' [3805](https://github.com/keptn/keptn/issues/3805)
+
+</p>
+</details>
+
+## Upgrade to 0.8.2
+
+- The upgrade from 0.8.x to 0.8.2 is supported by the `keptn upgrade` command. Find the documentation here: [Upgrade from Keptn 0.8.x to 0.8.2](https://keptn.sh/docs/0.8.x/operate/upgrade/#upgrade-from-keptn-0-8-1-to-0-8-2)
```

Comment:
The title says "Upgrade from Keptn 0.8.x to 0.8.2", but the link is `https://keptn.sh/docs/0.8.x/operate/upgrade/#upgrade-from-keptn-_0-8-1_-to-0-8-2`. Should we make that equal?